### DISCUSSION TITLE
Updating the table with many to many relations [May 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2146] [Discord Threads]
> 🙋‍♂️ **AK**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Please provide more details/steps on this.  
#  
> 🙋‍♂️ **AK**  
#### Likw what? Let me know what info you want / need and I'll give you 😀  
#  
> 💁 **o1lab**  
#### Like steps and errors pls  
#  
> 🙋‍♂️ **AK**  
#### Steps:   
#### Making api request to update row and empty the data in the related column from the table have the relation column.    
####    
#### Making api request to update row and empty the data in the related column from the related table    
#### (When you make relation it's making column in the related table)   
#### Tried to fetch the table to see what the name of the column to make sure I'm correct, I saw also a system column so I tried update it.   
#### Errors:   
#### If I will try to update only this column it will say the request data is empty (means not recognizing the column)   
#### Not only the relation one, not the system one and not the one in the related table. Error codr 400. If you want me to get the exect err msg I will get it.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/975488427644579931)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
